week-2/extended-project/valeria_and_friends/jsons/planets.json

```diff
@@ -0,0 +1,208 @@
+      "surface_water": "12",
+      "population": "4500000000",
+      "residents": [
+        "https://swapi.co/api/people/3/",
+        "https://swapi.co/api/people/21/",
+        "https://swapi.co/api/people/36/",
+        "https://swapi.co/api/people/37/",
+        "https://swapi.co/api/people/38/",
+        "https://swapi.co/api/people/39/",
+        "https://swapi.co/api/people/42/",
+        "https://swapi.co/api/people/60/",
+        "https://swapi.co/api/people/61/",
+        "https://swapi.co/api/people/66/",
+        "https://swapi.co/api/people/35/"
+      ],
+      "films": [
+        "https://swapi.co/api/films/5/",
+        "https://swapi.co/api/films/4/",
+        "https://swapi.co/api/films/6/",
+        "https://swapi.co/api/films/3/"
+      ],
+      "created": "2014-12-10T11:52:31.066000Z",
+      "edited": "2014-12-20T20:58:18.430000Z",
+      "url": "https://swapi.co/api/planets/8/"
+    },
+    {
+      "name": "Coruscant",
+      "rotation_period": "24",
+      "orbital_period": "368",
+      "diameter": "12240",
+      "climate": "temperate",
+      "gravity": "1 standard",
+      "terrain": "cityscape, mountains",
+      "surface_water": "unknown",
+      "population": "1000000000000",
+      "residents": [
+        "https://swapi.co/api/people/34/",
+        "https://swapi.co/api/people/55/",
+        "https://swapi.co/api/people/74/"
+      ],
+      "films": [
+        "https://swapi.co/api/films/5/",
+        "https://swapi.co/api/films/4/",
+        "https://swapi.co/api/films/6/",
+        "https://swapi.co/api/films/3/"
+      ],
+      "created": "2014-12-10T11:54:13.921000Z",
+      "edited": "2014-12-20T20:58:18.432000Z",
+      "url": "https://swapi.co/api/planets/9/"
+    },
+    {
+      "name": "Kamino",
+      "rotation_period": "27",
+      "orbital_period": "463",
+      "diameter": "19720",
+      "climate": "temperate",
+      "gravity": "1 standard",
+      "terrain": "ocean",
+      "surface_water": "100",
+      "population": "1000000000",
+      "residents": [
+        "https://swapi.co/api/people/22/",
+        "https://swapi.co/api/people/72/",
+        "https://swapi.co/api/people/73/"
+      ],
+      "films": ["https://swapi.co/api/films/5/"],
+      "created": "2014-12-10T12:45:06.577000Z",
+      "edited": "2014-12-20T20:58:18.434000Z",
+      "url": "https://swapi.co/api/planets/10/"
+    },
+    {
+      "name": "Geonosis",
+      "rotation_period": "30",
+      "orbital_period": "256",
+      "diameter": "11370",
+      "climate": "temperate, arid",
+      "gravity": "0.9 standard",
+      "terrain": "rock, desert, mountain, barren",
+      "surface_water": "5",
+      "population": "100000000000",
+      "residents": ["https://swapi.co/api/people/63/"],
+      "films": ["https://swapi.co/api/films/5/"],
+      "created": "2014-12-10T12:47:22.350000Z",
+      "edited": "2014-12-20T20:58:18.437000Z",
+      "url": "https://swapi.co/api/planets/11/"
+    }
+  ]
+}
```

Comment:
The same of https://github.com/ableco/codeable-exercises/pull/26/files#r264268573